src/pages/ProfilePage.js

```diff
@@ -0,0 +1,228 @@
+        const login = lodashGet(details, 'login', '');
+        const timezone = lodashGet(details, 'timezone', {});
+
+        // If we have a reportID param this means that we
+        // arrived here via the ParticipantsPage and should be allowed to navigate back to it
+        const shouldShowBackButton = Boolean(reportID);
+        const shouldShowLocalTime = !ReportUtils.hasAutomatedExpensifyEmails([login]) && !_.isEmpty(timezone);
+
+        let pronouns = lodashGet(details, 'pronouns', '');
+        if (pronouns && pronouns.startsWith(CONST.PRONOUNS.PREFIX)) {
+            const localeKey = pronouns.replace(CONST.PRONOUNS.PREFIX, '');
+            pronouns = this.props.translate(`pronouns.${localeKey}`);
+        }
+
+        const isSMSLogin = Str.isSMSLogin(login);
+        const phoneNumber = getPhoneNumber(details);
+        const phoneOrEmail = isSMSLogin ? getPhoneNumber(details) : login;
+
+        const isCurrentUser = _.keys(this.props.loginList).includes(login);
+
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title={this.props.translate('common.profile')}
+                    shouldShowBackButton={shouldShowBackButton}
+                    onBackButtonPress={() => Navigation.goBack()}
+                    onCloseButtonPress={() => Navigation.dismissModal()}
+                />
+                <View
+                    pointerEvents="box-none"
+                    style={[styles.containerWithSpaceBetween]}
+                >
+                    {_.isEmpty(details) ? (
+                        <FullScreenLoadingIndicator style={styles.flex1} />
+                    ) : (
+                        <ScrollView>
+                            <View style={styles.avatarSectionWrapper}>
+                                <AttachmentModal
+                                    headerTitle={displayName}
+                                    source={UserUtils.getFullSizeAvatar(avatar, login || accountID)}
+                                    isAuthTokenRequired
+                                    originalFileName={originalFileName}
+                                >
+                                    {({show}) => (
+                                        <PressableWithoutFocus
+                                            style={styles.noOutline}
+                                            onPress={show}
+                                        >
+                                            <OfflineWithFeedback pendingAction={lodashGet(details, 'pendingFields.avatar', null)}>
+                                                <Avatar
+                                                    containerStyles={[styles.avatarLarge, styles.mb3]}
+                                                    imageStyles={[styles.avatarLarge]}
+                                                    source={UserUtils.getAvatar(avatar, login || accountID)}
```

Comment:
Yeah this probably works because Concierge is always in their `personalDetailsList`. If it wasn't for some reason, maybe it wouldn't work? But I don't think that's a scenario we need to worry about since it shouldn't be possible.

Reply:
aah niiiiicie